test/ibmq/test_proxies.py

```diff
@@ -37,15 +35,15 @@
 INVALID_ADDRESS_PROXIES = {'https': '{}:{}'.format('invalid', PORT)}
 
 
-@skipIf(sys.version_info >= (3, 7), 'pproxy version not supported in 3.7')
+@skipIf(python_version < (2, 7), 'pproxy is supported only on or above python 2.7')
 class TestProxies(IBMQTestCase):
     """Tests for proxy capabilities."""
 
     def setUp(self):
         super().setUp()
-
+        listen_flag = '-i' if python_version <= (3, 5) else '-l'
```

Comment:
For maintainability, can you make this `if` based on the pproxy version instead of the python version?